.github/workflows/run-kata-deploy-tests-on-garm.yaml

```diff
@@ -34,6 +34,14 @@ jobs:
           - k0s
           - k3s
           - rke2
+        # TODO: include back when https://github.com/kata-containers/kata-containers/issues/9854 is fixed
+        exclude:
+          - vmm: clh
+            k8s: k0s
+          - vmm: clh
+            k8s: rke2
+          - vmm: qemu
+            k8s: k0s
```

Comment:
Please, disable the whole kata-deploy tests for now.

Reply:
as requested, I've just disable all kata-deploy-tests on garm.